lib/errors.js

```diff
@@ -334,7 +334,18 @@ function volapiErrorWrap(cause, message) {
     }
 }
 
+function DefaultFabricNetworkNotConfiguredError(cause) {
+    assert.optionalObject(cause, 'cause');
 
+    var errMsg = 'default_network is not configured for account';
+
+    _FriendlySigRestError.call(this, cause, errMsg);
```

Comment:
I realize you didn't add this, but this function name seems pretty odd. :) Nothing to change. Just pointing it out for the sake of conversation.